docs/src/main/asciidoc/native-and-ssl-guide.adoc

```diff
@@ -65,9 +65,17 @@ They enable the native SSL support for your native executable.
 
 As SSL is ipso facto the standard nowadays, we decided to enable its support automatically for some of our extensions:
 
- * the SmallRye REST client extension,
- * the Agroal connection pooling extension,
- * the Jaeger extension.
+ * the Agroal connection pooling extension (`quarkus-agroal`),
+ * the DynamoDB extension (`quarkus-amazon-dynamodb`),
+ * the Elasticsearch Rest client extension (`quarkus-elasticsearch-rest-client`),
```

Comment:
As said in another comment, here I would advertise the Hibernate Search + Elasticsearch extension instead of the client.

For now, we don't really want the users to include this extension directly.